### PR TITLE
chore: update README and logo

### PR DESCRIPTION
**Pull request by huijing**
## Changes proposed in this pull request

- This PR refines the elevator pitch of what Open Payments is.
- It also updates to the new Open Payments logo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,7 @@
 
 ## What is Open Payments?
 
-Open Payments is a protocol that can be implemented by digital wallets to
-facilitate interoperability in the setup of payments for different use cases
-including:
+Open Payments is an open API standard that can be implemented by account servicing entities to facilitate interoperability in the setup and completion of payments for different use cases including:
```
Comment (mkurapov) [inline README.md:9]:
```suggestion
Open Payments is an open API standard that can be implemented by account servicing entities (e.g. banks, digital wallet providers, and mobile money providers) to facilitate interoperability in the setup and completion of payments for different use cases including:
```
Maybe worth adding?

File: README.md [hunk 2]
```diff
@@ -6,9 +6,7 @@
```
Comment (mkurapov) [inline README.md:6]: While we are here, can we get rid of L32 (explaining about the landing page)

Comment (mkurapov) [inline README.md:6]: @sabineschaller this clock change stuff always messes with me, but is 1100 GMT still correct? I believe if the meeting is 1300 CET, then it should be 1200GMT

File: docs/public/favicon.svg
Comment (melissahenderson) [inline docs/public/favicon.svg:1]: Is there a dark mode version of this file? In dark mode the middle circle is still white.... unless that's intentional.

**Pull request merged**